stdlib/os/__init__.pyi

```diff
@@ -365,7 +365,8 @@ class stat_result(structseq[float], tuple[int, int, int, int, int, int, int, flo
     if sys.version_info >= (3, 12) and sys.platform == "win32":
         @property
         @deprecated(
-            "Use st_birthtime instead to retrieve the file creation time. In the future, this property will contain the last metadata change time."
+            """Use st_birthtime instead to retrieve the file creation time. \
```

Comment:
This may cause type checkers to add spurious whitespace in the middle of the deprecation message.

```
>>> inspect.cleandoc("""Use st_birthtime instead to retrieve the file creation time. \
...             In the future, this property will contain the last metadata change time.""")
'Use st_birthtime instead to retrieve the file creation time.             In the future, this property will contain the last metadata change time.'
```

(PEP 702 recommends that type checkers use `inspect.cleandoc` or equivalent logic to clean up deprecation messages.)

Reply:
My bad, I forgot about extra whitespaces, thanks for catching it !